lib/bolt_server/file_cache.rb

```diff
@@ -19,38 +27,56 @@ def tmppath
     def setup
       FileUtils.mkdir_p(@cache_dir)
       FileUtils.mkdir_p(tmppath)
+      self
     end
 
-    # TODO: I feel like we shouldn't drop down to this level
-    # faraday is alread a dependency should we use that?
-    # TODO: timeouts? proxies?
-    def request_file(path, params, file)
-      # TODO: handle trailing /
-      uri = "#{@config.file_server_uri}#{path}"
+    def ssl_cert
+      @ssl_cert ||= File.read(@config.ssl_cert)
+    end
 
+    def ssl_key
+      @ssl_key ||= File.read(@config.ssl_key)
+    end
+
+    def client
```

Comment:
I 'm not sure what http library we'd want to use but I think this should keep the connection open for 2s by default.